### PR TITLE
Update containers.md

### PR DESCRIPTION
**Pull request by ring-ring-ring**
removed mention of Celery as requested by @praseodym  https://github.com/minvws/nl-kat-coordination/issues/808#issuecomment-1527394169

### CODE REVIEW

File: docs/source/technical_design/containers.md [hunk 1]
```diff
@@ -102,8 +102,7 @@ We have two container images that are used to run multiple containers. What the
 | boefjes         | normalizer  | Normalizers runtime                                                               |
 | boefjes         | katalogus   | Katalogus API                                                                     |
 | octopoes        | web         | Octopoes API                                                                      |
-| octopoes        | worker-beat | Celery worker running beat. There must only be exactly one container of this type |
```
Comment (ammar92) [inline docs/source/technical_design/containers.md:105]: I think this one still exists in the current Compose project setup

File: docs/source/technical_design/containers.md [hunk 2]
```diff
@@ -102,8 +102,7 @@ We have two container images that are used to run multiple containers. What the
 | boefjes         | normalizer  | Normalizers runtime                                                               |
 | boefjes         | katalogus   | Katalogus API                                                                     |
 | octopoes        | web         | Octopoes API                                                                      |
```
Comment (praseodym) [inline docs/source/technical_design/containers.md:104]: This container is called `octopoes_api` now

File: docs/source/technical_design/containers.md (hunk 1 above)
Comment (praseodym) [reply docs/source/technical_design/containers.md:105]: The container is called `octopoes_api_worker` and it processes events from RabbitMQ. We no longer use Celery.

**Review by praseodym — CHANGES_REQUESTED** [review]

File: docs/source/technical_design/containers.md (hunk 1 above)
Comment (ammar92) [reply docs/source/technical_design/containers.md:105]: Yes the service has been renamed but there are still references to celery (as a dependency and code)

**Comment by ring-ring-ring** [comment]
closed after short talk - we solve this in a different way

**Pull request closed without merging**